### ISSUE TITLE
Incompatible with airflow 2.7.x due to jsonschema version conflict

### ISSUE THREAD

**Issue by KulykDmytro** [datahub-project/datahub#8859]
**Describe the bug**
`jsonschema` library version conflict when building a poerty project with `airflow:2.7.1` and `datahub:0.11`
is it possible to loosen version dependencies

**To Reproduce**
* pyproject.toml
```toml
[tool.poetry]
name = "airflow-datahub"
version = "0.1.0"

[tool.poetry.dependencies]
python = "~3.10"

apache-airflow = { version = "2.7.1", extras = [ "statsd" ] }
acryl-datahub = { version = "^0.11", extras = [
    "datahub-rest", "postgres", "kafka", "kafka-connect", "ldap", "tableau", "superset",
    "athena", "glue", "mysql"
] }

[build-system]
requires = ["poetry-core>=1.0.0"]
build-backend = "poetry.core.masonry.api"
```
* `poetry update --lock`

**Screenshots**
```
Because apache-airflow (2.7.1) depends on jsonschema (>=4.18.0)
   and acryl-datahub (0.11.0) depends on jsonschema (<=4.17.3), apache-airflow (2.7.1) is incompatible with acryl-datahub (0.11.0).
  And because no versions of acryl-datahub match >0.11.0,<0.12.0, apache-airflow (2.7.1) is incompatible with acryl-datahub (>=0.11.0,<0.12.0).
  So, because airflow-datahub depends on both apache-airflow (2.7.1) and acryl-datahub (^0.11.0), version solving failed.
```

**Additional context**
poetry:1.4.2
python: 3.10
os: MacOS

**Comment by hsheth2** [comment]
@KulykDmytro we did this a few days ago: https://github.com/datahub-project/datahub/pull/8842. It will be in a release soon.

Closing this issue for now, but let us know if you continue to have issues with it.

**Comment by viniciusdsmello** [comment]
Hi @hsheth2, do you know when the next release is coming?

**Comment by hsheth2** [comment]
It's available in acryl-datahub v0.11.0.1
